subworkflows/local/recalibrate/main.nf

```diff
@@ -11,54 +11,49 @@ include { SAMTOOLS_STATS               } from '../../../modules/nf-core/samtools
 workflow RECALIBRATE {
     take:
         skip_samtools  // boolean: true/false
-        bam            // channel: [mandatory] bam
-        dict           // channel: [mandatory] dict
-        fai            // channel: [mandatory] fai
-        fasta          // channel: [mandatory] fasta
+        ch_bam         // channel: [mandatory] bam
+        ch_dict        // channel: [mandatory] dict
+        ch_fai         // channel: [mandatory] fai
+        ch_fasta       // channel: [mandatory] fasta
```

Comment:
I don't like the `ch_` prefixes

Reply:
but fine by me

Reply:
Yeah me neither but I had to do it and it was the best solution in this case. The language server was complaining about `map` operator named variables that had the same name as the input channels. Feel free to suggest a better name in this case :)

Reply:
I use the _ suffix in map for this case

Reply:
Hmmm okay good idea! I'll update it